moveit_core/constraint_samplers/src/default_constraint_samplers.cpp

```diff
@@ -565,6 +555,26 @@ bool constraint_samplers::IKConstraintSampler::sampleHelper(robot_state::RobotSt
       return false;
     }
 
+    // we now have the transform we wish to perform IK for, in the planning frame
+    if (transform_ik_)
+    {
+      // we need to convert this transform to the frame expected by the IK solver
+      // both the planning frame and the frame for the IK are assumed to be robot links
```

Comment:
@v4hn right!  I get that each IK plugin may define a different reference frame which is usually the first (bottom) link in the kinematic chain.  My comments was more in regards to whether saying `... IK are assumed to be robot links` conveys that idea properly.